inbm/dispatcher-agent/dispatcher/source/constants.py

```diff
@@ -0,0 +1,49 @@
+"""
+    Copyright (C) 2023 Intel Corporation
+    SPDX-License-Identifier: Apache-2.0
+"""
+
+
+from enum import Enum, unique
+from dataclasses import dataclass, field
+from typing import List
+
+UBUNTU_APT_SOURCES_LIST = "/etc/apt/sources.list"
+UBUNTU_APT_SOURCES_LIST_D = "/etc/apt/sources.list.d"
+
+
+@dataclass(kw_only=True, frozen=True)
+class ApplicationSourceList:
+    name: str
+    sources: list[str]
+
+
```

Comment:
Are these dataclasses still used?

Reply:
I think I see where they are used now.  They didn't come up in a search, but I do see them being used.